sdk/polling_manager.ts

```diff
@@ -0,0 +1,29 @@
+import { Flagsmith } from '.';
+
+export class EnvironmentDataPollingManager {
+    private interval?: NodeJS.Timer;
+    private main: Flagsmith;
+    private refreshIntervalSeconds: number;
+
+    constructor(main: Flagsmith, refreshIntervalSeconds: number) {
+        this.main = main;
+        this.refreshIntervalSeconds = refreshIntervalSeconds;
+    }
+
+    start() {
+        const updateEnvironemnt = () => {
+            this.interval = setInterval(async () => {
+                await this.main.update_environment();
+            }, this.refreshIntervalSeconds * 1000);
+        };
+
+        this.interval = setTimeout(updateEnvironemnt, this.refreshIntervalSeconds * 1000);
```

Comment:
This assignment doesn't do anything, also it's waiting for the refreshIntervalSeconds to then wait again the refreshIntervalSeconds inside updateEnvironemnt